app/src/main/java/org/p2p/wallet/main/ui/send/SendPresenter.kt

```diff
@@ -365,11 +369,17 @@ class SendPresenter(
         val isEnabled = isNotZero && !isMoreThanBalance
 
         val isValidAddress = isAddressValid(destinationAddress)
+        val isUserNameValid = isUsernameValid(username)
         val availableColor = if (isMoreThanBalance) R.attr.colorAccentWarning else R.attr.colorAccentPrimary
         view?.setAvailableTextColor(availableColor)
-        view?.showButtonEnabled(isEnabled && isValidAddress && !shouldAskConfirmation)
+        view?.showButtonEnabled(
+            isEnabled && isValidAddress || isUserNameValid && !shouldAskConfirmation
```

Comment:
Здесь кажется ты сломаешь логику. 
```suggestion
            isEnabled && (isValidAddress || isUserNameValid) && !shouldAskConfirmation
```